### ISSUE TITLE
Provide additional accessibility module that we want to include by default (e.g. everything except grid selection on element creation, focus tracker and toast messages) and enable it as part of the defaults

### ISSUE THREAD

**Issue by ndoschek** [eclipse-glsp/glsp#1128]

**Comment by tortmayr** [comment]
I also had a discussion with Philip regarding this topic. IMO when it comes to accessibility all related features must be included by default. It does not mean the that all features of the accessibility module are active by default. Nevertheless it should be possible to activate them on the fly during runtime. With the approach you described, if a user wants to use an accessibility feature that is not part of the default accessibility module it would be technically required to recompile the editor and load the additional accessibility features. That`s not really in the spirit of accessibility 😉 

So selection on element creation, focus tracker and toast messages should be included in the default accessibility module but are simply not active per default. User can then activate them via keyboard shortcut/menu entries.

**Comment by ndoschek** [comment]
Sure, feel free to adapt the description, I just copied the suggestions from the initial issues over to separate follow-ups. Thanks!
